test/connectors/CompanyHouseConnectorSpec.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.SpecBase
+import models.{Submission, SubmissionResponse}
+import org.scalatest.concurrent.ScalaFutures
+import play.api.libs.json.{JsValue, Json}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse, UpstreamErrorResponse}
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
```

Comment:
```suggestion
import uk.gov.hmrc.http.HttpClient
```
This clears out some compiler warnings

Reply:
Also there's a few unused imports that can be cleaned up